gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/CopySource.java

```diff
@@ -112,11 +116,21 @@
   public static final String SIMULATE = CopyConfiguration.COPY_PREFIX + ".simulate";
   public static final String MAX_SIZE_MULTI_WORKUNITS = CopyConfiguration.COPY_PREFIX + ".binPacking.maxSizePerBin";
   public static final String MAX_WORK_UNITS_PER_BIN = CopyConfiguration.COPY_PREFIX + ".binPacking.maxWorkUnitsPerBin";
+  public static final String REQUESTS_EXCEEDING_AVAILABLE_RESOURCE_POOL_EVENT_NAME =
+      "RequestsExceedingAvailableResourcePoolEvent";
+  public static final String REQUESTS_DROPPED_EVENT_NAME = "RequestsDroppedEvent";
+  public static final String REQUESTS_REJECTED_DUE_TO_INSUFFICIENT_EVICTION_EVENT_NAME =
+      "RequestsRejectedDueToInsufficientEvictionEvent";
+  public static final String REQUESTS_REJECTED_WITH_LOW_PRIORITY_EVENT_NAME = "RequestsRejectedWithLowPriorityEvent";
+  public static final String FILESET_NAME = "fileset.name";
+  public static final String FILESET_TOTAL_ENTITIES = "fileset.total.entities";
+  public static final String FILESET_TOTAL_SIZE_IN_BYTES = "fileset.total.size";
 
   private static final String WORK_UNIT_WEIGHT = CopyConfiguration.COPY_PREFIX + ".workUnitWeight";
   private final WorkUnitWeighter weighter = new FieldWeighter(WORK_UNIT_WEIGHT);
 
   public MetricContext metricContext;
+  public EventSubmitter _eventSubmitter;
```

Comment:
Please be consistent on variable naming within a class. This class does not use underscores.